staging/src/k8s.io/kubectl/pkg/util/prune/prune_test.go

```diff
@@ -48,23 +48,48 @@ func (m *testRESTMapper) RESTMapping(gk schema.GroupKind, versions ...string) (*
 
 func TestGetRESTMappings(t *testing.T) {
 	tests := []struct {
-		mapper      *testRESTMapper
-		pr          []Resource
-		expectedns  int
-		expectednns int
-		expectederr error
+		mapper             *testRESTMapper
+		pr                 []Resource
+		namespaceSpecified bool
+		expectedns         int
+		expectednns        int
+		expectederr        error
 	}{
 		{
-			mapper:      &testRESTMapper{},
-			pr:          []Resource{},
+			mapper:             &testRESTMapper{},
+			pr:                 []Resource{},
+			namespaceSpecified: false,
+			expectedns:         14,
+			expectednns:        2,
+			expectederr:        nil,
+		},
+		{
+			mapper:             &testRESTMapper{},
+			pr:                 []Resource{},
+			namespaceSpecified: true,
+			// it will be 12 namespaced resources and 0 non-namespaced resources after the deprecation period has passed.
```

Comment:
why would the number of namespaced resources decrease?

Reply:
It is my fault. 
In the future,
```
			expectedns:  14,
			expectednns: 0,
```
It should be like above.

Reply:
Updated.